docs/cudf/source/basics/basics.rst

```diff
@@ -15,32 +15,61 @@ The following table lists all of cudf types. For methods requiring dtype argumen
+    | (duration type)        |                              |                                                                                     | ``'timedelta64[us]'``, ``'timedelta64[ns]'``|
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | Categorical            | CategoricalDtype             | (none)                                                                              | ``'category'``                              |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | Boolean                |                              | np.bool_                                                                            | ``'bool'``                                  |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | Decimal                | Decimal64Dtype               | (none)                                                                              | (none)                                      |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | List                   | ListDtype₁.₁                 | np.list_                                                                            | (none):sub:``1.2``                          |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | Struct                 | StructDtype₁.₁               | np.struct_                                                                          | (none):sub:``1.2``                          |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
 
 **Note: All dtypes above are Nullable**
 
+**1.1 The complete datatypes for both lists and structs are inferred from the data that it is composed upon. For example, the data type for the value** ``[[1, 2], [3, 4]]`` **is inferred as** ``ListDtype(int64)``.
+
+**1.2 cuDF does not support string aliases where dtype would equal** ``'list'`` **or** ``'struct'`` **list's and structs are inferred by the datatype explicitly or through a determination based on the data that the list or struct is composed upon**
+
+**cuDF supports arbitrarily deep nested lists. Such as** ``list(list(int))``, **even list of structs or structs of lists**
```

Comment:
It feels like this section marks a change from footnotes to details about struct and list types. This might be a good spot for a sub-header and a shift back to standard format (non-bold).

Reply:
Will be resolved in the next commit. Will begin new sub header self described as 
Struct and List datatypes
cuDF supports arbitrarily deep...